gov2/bedrock-runtime/README.md

```diff
@@ -39,11 +38,12 @@ For prerequisites, see the [README](../README.md#Prerequisites) in the `gov2` fo
 
 Code excerpts that show you how to call individual service functions.
 
-- [AI21 Labs Jurassic-2: Text generation](actions/invoke_model.go#L78) (`InvokeModel`)
-- [Amazon Titan: Image generation](actions/invoke_model.go#L178) (`InvokeModel`)
-- [Anthropic Claude 2: Real-time response stream processing](actions/invoke_model_with_response_stream.go#L30) (`InvokeModelWithResponseStream`)
-- [Anthropic Claude 2: Text generation](actions/invoke_model.go#L27) (`InvokeModel`)
-- [Meta Llama 2: Text generation](actions/invoke_model.go#L130) (`InvokeModel`)
+- [Image generation with Amazon Titan Image Generator G1](actions/invoke_model.go#L191) (`InvokeModel`)
```

Comment:
This format does not pass our validator. See the failing checks.
To fix this, follow the instructions in `.tools/readmes` to run `multi.py`. Other READMEs that are not this one may be modified. Just discard those and keep the changes for this one.

Reply:
Fixed.